Makefile

```diff
@@ -6,35 +6,35 @@ VERSION_NAME=v1.0.0
 lint:
 	black src/
 	flake8 src/
-.phony: lint
+.PHONY: lint
```

Comment:
why all caps this?

Reply:
that's the syntax (https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html). otherwise it doesn't work